scripts/e2e/bootstrap_job/spec/machineset.template

```diff
@@ -0,0 +1,35 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineSet
+metadata:
+  name: clusterapi-machineset-1
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      node-type: worker-node
+  template:
+    metadata:
+      labels:
+        node-type: worker-node
+    spec:
+      providerConfig:
+        value:
+          apiVersion: "vsphereproviderconfig/v1alpha1"
+          kind: "VsphereMachineProviderConfig"
+          vsphereMachine: "standard-node"
+          machineVariables:
+            datacenter: "SDDC-Datacenter"
+            datastore: "WorkloadDatastore"
+            resource_pool: "clusterapi"
+            vm_folder: "clusterapi"
+            network: "sddc-cgw-network-3"
+            num_cpus: "2"
+            memory: "2048"
+            vm_template: "ubuntu-16.04-server-cloudimg-amd64"
+            disk_label: "Hard disk 1"
+            disk_size: "15"
+            virtual_machine_domain: ""
+      versions:
+        kubelet: 1.10.1
```

Comment:
For consistency, I think we should use the same version as above 1.11.2.

Reply:
fixed